docs: Solana Vault swaps upgrade and CCM length update for 1.9.2 (#570)
chainflip-io/chainflip-docs: pages/swapping/integrations/javascript-sdk/swap-assets/encode-vault-swap-data.mdx

            Additional data that is required for encoding vault swap transaction data for specific chains.
            <ul>
                <li>

>                     For swaps starting from Solana, <a href="../../advanced/vault-swaps/solana#user-created-event-data-account"><code>seed</code></a> can optionally be provided.
>                     When a seed is not provided a random one will be generated on the fly.

Reviewer: ```suggestion
                    For swaps starting from Solana, a <a href="../../advanced/vault-swaps/solana#user-created-event-data-account"><code>seed</code></a> can be set optionally.
                    If no seed is set, a random one will be generated on the fly.
```
